content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -199,6 +199,12 @@ Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-pl
 or stand-alone.  In either version, it doesn't require any configuration or extra code
 to run, and in both cases, the network provides one IP address per pod - as is standard for Kubernetes.
 
+### How to apply weave-net in kubernetes cluster 
+
+```
+kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
+```
+
```

Comment:
This detail belongs on the Weave Net website and not in the Kubernetes docs.
We wouldn't accept this change @mansurali901